sdk/tests/deqp/framework/opengl/gluDrawUtil.js

```diff
@@ -40,7 +40,7 @@ var gluShaderProgram = framework.opengl.gluShaderProgram;
  */
 gluDrawUtil.VertexArrayBinding = function(type, location, components, elements, data, stride, offset) {
     this.type = type;
-    this.location = location === undefined ? -1 : location;
+    this.location = location === undefined ? 0 : location;
```

Comment:
You are correct, ken.  In gluDrawUtil.bindingPointFromName = function(name, location), location may be undefined, we should initialize it to 0. 

For Mo's concern, I didn't find the undefined case for location argument in gluDrawUtil.VertexArrayBinding. Should we keep it as before or change to "this.location = location;"?